src/ctapipe_io_lst/tests/test_lsteventsource.py

```diff
@@ -450,4 +450,4 @@ def test_arbitrary_filename(tmp_path):
         n_read = 0
         for _ in source:
             n_read += 1
-        assert n_read == n_read
+        assert n_read == 50
```

Comment:
Is this the number of events in the file above (I guess newly added to the test dataset)?

Reply:
It's not a new file, I just copied the existing test file into a file with a different name.

But yes, we know that file has 50 events